k6ext/k6test/vu.go

```diff
@@ -10,16 +9,21 @@ import (
 	k6eventloop "go.k6.io/k6/js/eventloop"
 	k6modulestest "go.k6.io/k6/js/modulestest"
 	k6lib "go.k6.io/k6/lib"
+	k6testutils "go.k6.io/k6/lib/testutils"
 	k6metrics "go.k6.io/k6/metrics"
 
 	"github.com/dop251/goja"
 	"github.com/oxtoacart/bpool"
-	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/guregu/null.v3"
 )
 
 // VU is a k6 VU instance.
+// TODO: Do we still need this VU wrapper?
+// ToGojaValue can be a helper function that takes a goja.Runtime (although it's
+// not much of a helper from calling ToValue(i) directly...), and we can access
+// EventLoop from modulestest.Runtime.EventLoop. I guess we still need the
+// RunLoop() override to call WaitOnRegistered()?
```

Comment:
I would argue that k6 APIs (especially the new ones like `modulestest`) are in flux, and we better keep the wrapper and other helpers. Also, the helpers provide easy-to-remember (at least, for me) names for the common and mostly verbose operations.

Reply:
1. I would argue you should not have `WaitOnRegistered` *before* the `LoopStart` you might want to have it after to wait on everything to finish in case of errors. But adding it before just masks that a previous test didn't do something properly and you are reusing the same loop.
2. You can still just wrap `modulestest.Runtime` instead of VU ... but maybe you will need to rename the struct ;). And if you embed modulestest.Runtime (or the wrapper) in testBrowser, you should need to touch as many places to rename the access to `VU` due to how embedding fields in structs works.

Reply:
1. Yeah, I was suspicious of adding `WaitOnRegistered` there. The problem is in the tests (`slow_mo` tests) that share the same browser and VU.
2. I guess `modulestest.Runtime` is added in k6@v0.4.0 and we can take advantage of it 👍